lib/orchestrator_client/command.rb

```diff
@@ -4,18 +4,33 @@ def initialize(https)
     @https = https
   end
 
-  def task(options = {})
+  def deploy(options = {})
     raise ArgumentError, 'Must pass options as a hash' unless options.is_a? Hash
-    @https.post("command/task", options)
+    @https.post("command/deploy", options)
   end
 
-  def deploy(options = {})
+  def plan_start(options = {})
```

Comment:
When will plan_start/plan_finish be used? Or how are they exposed?

Reply:
`client.command.plan_start`

Reply:
🤷‍♂️

Reply:
bolt will call it directly in other words. The orchestrator transport is informed that a plan is running before the plan starts. It then calls start plan on the client before running any tasks. `finish_task` is called after the plan finishes and orchestrator calls finish_task on any clients that were used during the plan.